docs/release.md

```diff
@@ -29,12 +29,16 @@ make container-push
 make bundle-publish
 ```
 
+A real example of an alpha release can be found at [release: v1.8.0-alpha.1](https://github.com/3scale-ops/cert-manager-helm-operator/commit/e3ac092d7c07fd3ca631dcd2418ea578a3c10221) commit.
+
 ## Stable
 
 * Execute the following target to create appropiate `alpha` and `stable` bundle files:
 
 ```bash
 make prepare-stable-release
 ```
+A real example of an alpha release can be found at [release: v1.8.0-alpha.1](https://github.com/3scale-ops/cert-manager-helm-operator/commit/e3ac092d7c07fd3ca631dcd2418ea578a3c10221) commit.
```

Comment:
I guess here you wanted to add a commit for stable instead oc copy/paste previous alpha??

Reply:
Yep, was accidentally left outside the commit.